docs/dev/table/tableApi.md

```diff
@@ -1682,6 +1682,36 @@ The `OverWindow` defines a range of rows over which aggregates are computed. `Ov
 
 {% top %}
 
+### Aggregate
+
+Aggregate performs an aggregate operation with an aggregate function. You have to close the
+"aggregate" with a select statement. The output will be flattened if the output type is a
+composite type.
+
+<div class="codetabs" markdown="1">
+<div data-lang="java" markdown="1">
+{% highlight java %}
+tableEnv.registerFunction("myAggFunc", new MyAggregateFunction());
```

Comment:
How about info user that `MyAggregateFunction` is an `AggregateFunction` by define a variable: `AggregateFunction<?,?> myAggFunc = new MyAggregateFunction()`. Same as the scala example. We will have Table aggregate functions later. I think it would be nice if we distinguish them.

Reply:
Done